qa-tests-backend/src/test/groovy/NetworkBaselineTest.groovy

```diff
@@ -297,59 +307,58 @@ class NetworkBaselineTest extends BaseSpecification {
         )
         validateBaseline(postLockClientBaseline, beforeDeferredCreate, justAfterDeferredCreate,
             [], [])
-    }
 
-    @Unroll
-    // TODO: ROX-11126
-    @Ignore
-    @Category(NetworkBaseline)
-    def "Verify user get for non-existent baseline"() {
         when:
-        "Create initial set of deployments, wait for baseline to populate"
-        def beforeDeploymentCreate = System.currentTimeSeconds()
-        batchCreate([USER_DEP, BASELINED_USER_CLIENT_DEP])
-        def justAfterDeploymentCreate = System.currentTimeSeconds()
+        "Verify user get for non-existent baseline"
+        def beforeUserServiceDeploymentCreate = System.currentTimeSeconds()
+        batchCreate([USER_DEP])
+        def justAfterUserServiceDeploymentCreate = System.currentTimeSeconds()
 
-        def serverDeploymentID = USER_DEP.deploymentUid
-        assert serverDeploymentID != null
+        def userReqBaselineServerDeploymentID = USER_DEP.deploymentUid
+        assert userReqBaselineServerDeploymentID != null
 
-        def baselinedClientDeploymentID = BASELINED_USER_CLIENT_DEP.deploymentUid
-        assert baselinedClientDeploymentID != null
+        log.info "Deployment IDs Server: ${userReqBaselineServerDeploymentID}"
 
-        log.info "Deployment IDs Server: ${serverDeploymentID}, " +
-                    "Baselined client: ${baselinedClientDeploymentID}"
+        // Get the server baseline to simulate a user asking for a baseline prior to
+        // observation ending.  This will generate a baseline at the time of request
+        // instead of after observation.
+        assert NetworkBaselineService.getNetworkBaseline(userReqBaselineServerDeploymentID)
```

Comment:
Here we're just asserting that _something_ returns from `getNetworkBaseline`. Shouldn't we check some property of the baseline? E.g. you mention in the comment above about the observation ending, isn't it worth asserting on it?

Reply:
yeah that makes sense.